### PR TITLE
fix: submit enable when replace gpkg ( MAPCO-5944 )

### PR DESCRIPTION
**Pull request by Eliav-Maimon**

### CODE REVIEW

File: src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx [hunk 1]
```diff
@@ -71,33 +74,40 @@ const setDefaultValues = (record: Record<string, unknown>, descriptors: EntityDe
...
-      });
-      // record.updateDate = moment();
-      record.sensors = [];
-      record.productType = ProductType.ORTHOPHOTO;
-      record['__typename'] = LayerRasterRecordModel.properties['__typename'].name.replaceAll('"','');
-      break;
-    default:
-      break;
-  }
+  // switch (recordType) {
```
Comment (alebinson) [inline src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx:94]: U can remove **commented** block

File: src/discrete-layer/components/layer-details/layer-details.tsx [hunk 2]
```diff
@@ -49,6 +49,7 @@ import { PYCSW_ANY_TEXT_FIELD } from '../map-container/freeTextSearch.component'
 import { ResolutionValuePresentorComponent } from './field-value-presentors/resolution.value-presentor';
 
 const FOOTPRINT_FIELD_NAMES = ['footprint', 'geometry'];
+const DEFAULT_ENUM = 'DUMMY';
```
Comment (alebinson) [inline src/discrete-layer/components/layer-details/layer-details.tsx:52]: move it to /layer-details/utils.ts

File: src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx [hunk 3]
```diff
@@ -54,6 +56,7 @@ import './entity.raster.dialog.css';
 
 const IS_EDITABLE = 'isManuallyEditable';
 const DEFAULT_ID = 'DEFAULT_UI_ID';
+const DEFAULT_TYPE_NAME = 'DUMMY';
```
Comment (alebinson) [inline src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx:59]: move it to /layer-details/utils.ts

**Review by alebinson — CHANGES_REQUESTED** [review]
see my comments

File: src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx (hunk 3 above)
Comment (Eliav-Maimon) [reply src/discrete-layer/components/layer-details/raster/entity.raster.dialog.tsx:59]: Decided to live it as is

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**